.expeditor/scripts/shared.sh

```diff
@@ -50,12 +66,13 @@ install_latest_hab_binary() {
     local pkg_target="${1:-$BUILD_PKG_TARGET}"
     curlbash_hab "${pkg_target}"
 
-    hab_binary="/bin/hab"
-    # TODO: workaround for https://github.com/habitat-sh/habitat/issues/6771
-    ${hab_binary} pkg install core/hab-studio
+    if [ "$pkg_target" != "x86_64-darwin" ]; then
+        # TODO: workaround for https://github.com/habitat-sh/habitat/issues/6771	
+        ${hab_binary} pkg install core/hab-studio
+    fi
```

Comment:
There's actually no reason this function should be called on macOS, because it's really about ensuring that the hab and studio version that we're currently building are installed on the machine. Since that's explicitly not something that we ever do on macOS, we don't need this function.

I'd argue that we should probably error if this is called on macOS at the moment, just to help keep us honest.

Reply:
Good call - I didn't see the forest for the trees :D

Reply:
:palm_tree: :christmas_tree: :tanabata_tree: :deciduous_tree: :evergreen_tree: